packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Title.tsx

```diff
@@ -61,16 +62,21 @@ const ComponentContent = (props: ComponentContentProps) => {
 export const Default = (props: TitleProps): JSX.Element => {
   const datasource = props.fields?.data?.datasource || props.fields?.data?.contextItem;
   const { sitecoreContext } = useSitecoreContext();
+  const isMetadataMode = sitecoreContext?.editMode === 'metadata';
 
   const text: TextField = {
     value: datasource?.field?.jsonValue?.value,
-    editable: datasource?.field?.jsonValue?.editable,
+    ...(isMetadataMode
+      ? { metadata: props?.fields?.data?.contextItem?.field?.jsonValue?.metadata }
+      : { editable: datasource?.field?.jsonValue?.editable }),
   };
   const link: LinkField = {
     value: {
       href: datasource?.url?.path,
       title: datasource?.field?.jsonValue?.value,
-      editable: true,
+      ...(isMetadataMode
+        ? { metadata: props?.fields?.data?.contextItem?.field?.jsonValue?.metadata }
+        : { editable: true }),
```

Comment:
There is no `editable` on `LinkFieldValue` type (not sure how this would be doing anything currently). Both this and `metadata` should be a level up on `LinkField` if you want to make it inline editable. And it wouldn't be a boolean.